dataland-frontend/src/components/resources/newLandingPage/TheQuotes.vue

```diff
@@ -18,7 +18,16 @@
       <div v-for="(card, index) in cards" :key="index" role="listitem" class="quotes__slide">
         <div class="quotes__slide-videoContainer">
           <iframe
-            :src="'https://www.youtube.com/embed/' + card.icon + '?rel=0'"
+            class="cookieconsent-optout-marketing"
+            :src="'https://www.youtube-nocookie.com/embed/' + card.icon + '?rel=0'"
+            title="Youtube video player"
+            allowfullscreen
+            :class="{ 'quotes__slide-video--zoom-out': currentSlide !== index - 1, 'quotes__slide-video': true }"
+          ></iframe>
+          <iframe
+            class="cookieconsent-optin-marketing"
+            :data-src="'https://www.youtube.com/embed/' + card.icon + '?rel=0'"
+            data-cookieconsent="marketing"
```

Comment:
Does this work like a show-if? I'm surprised that both the no-cookie and the cookie iframe appear to be part of the page at all times.

Reply:
yes, that's how it works.
the css class is created by coookiebot.com
(If i wanted to do this with v-if i would need to ask for consent anyways and contact cookiebot for that)